docs/3.0-migration-guide.d/remove_supp_for_extensions_in_pre-v3_X_509_certs.md

```diff
@@ -0,0 +1,11 @@
+Remove the `MBEDTLS_X509_ALLOW_EXTENSIONS_NON_V3` option
+--
+
+This change does not affect users who are working with current V3 X.509
```

Comment:
Suggested addition: "This change does not affect people who were using the default configuration, as this option was already disabled by default. Also, it does not affect ..."

Reply:
Done